tests/RobotFramework/tests/cumulocity/registration/registration_lifecycle.robot

```diff
@@ -56,16 +56,36 @@ Register child device with defaults via MQTT
 Register child device with custom name and type via MQTT
     Execute Command
     ...    tedge mqtt pub --retain 'te/device/${CHILD_SN}//' '{"@type":"child-device","name":"${CHILD_SN}","type":"linux-device-Aböut"}'
+
     Should Have MQTT Messages
-    ...    te/device/${CHILD_SN}//
-    ...    message_contains="@id":"${CHILD_XID}"
-    ...    message_contains="@type":"child-device"
+    ...    te/device/${CHILD_SN}///twin/name
+    ...    message_contains=${CHILD_SN}
+    Should Have MQTT Messages
+    ...    te/device/${CHILD_SN}///twin/type
+    ...    message_contains=linux-device-Aböut
     Check Child Device
     ...    parent_sn=${DEVICE_SN}
     ...    child_sn=${CHILD_XID}
     ...    child_name=${CHILD_SN}
     ...    child_type=linux-device-Aböut
 
+Register child device with initial twin data
+    Execute Command
+    ...    tedge mqtt pub --retain 'te/device/${CHILD_SN}//' '{"@type": "child-device", "@id":"${CHILD_SN}", "name":"${CHILD_SN}", "maintenance_mode": true, "maintenance_window": 5}'
+
+    Should Have MQTT Messages
+    ...    te/device/${CHILD_SN}///twin/name
+    ...    message_contains=${CHILD_SN}
+    Should Have MQTT Messages
+    ...    te/device/${CHILD_SN}///twin/maintenance_mode
+    ...    message_contains=true
+    Should Have MQTT Messages
+    ...    te/device/${CHILD_SN}///twin/maintenance_window
+    ...    message_contains=5
+
+    ${foo}=    Device Should Exist    ${CHILD_SN}
```

Comment:
Is this variable used for anything? If not, then it should be deleted (as I'm sure the Robocop linter will complain about it).

Reply:
Oops. Resolved in 8eaef40.